doc/source/ray-core/doc_code/fault_tolerance_tips.py

```diff
@@ -0,0 +1,77 @@
+# __return_ray_put_start__
+import ray
+
+# Non-fault tolerant version:
+@ray.remote
+def A():
```

Comment:
```suggestion
def a():
```

Python convention is lower-case for functions.